plugins/org.jkiss.dbeaver.erd.ui/plugin.xml

```diff
@@ -21,6 +21,7 @@
 
     <extension-point id="org.jkiss.dbeaver.erd.ui.export.format" name="ERD Export Formats" schema="schema/org.jkiss.dbeaver.ext.erd.export.format.exsd"/>
     <extension-point id="org.jkiss.dbeaver.erd.ui.notation.style" name="ER Diagram Notation Style Specification" schema="schema/org.jkiss.dbeaver.erd.ui.notation.style.exsd"/>
+    <extension-point id="org.jkiss.dbeaver.erd.ui.routing" name="ER Diagram Connection Routing Specification" schema="schema/org.jkiss.dbeaver.erd.ui.routing.exsd"/>
```

Comment:
Maybe repurpose existing extension point instead creating new ones?

Reply:
Maybe, but require to modify "notation style" ext name to some generic and care of compatibility with user workspace

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/32031248/591240ed-5445-458b-94c6-d807fd3c492f)

Reply:
The new route provided  logic for bridges that adjust clarity of diagram.

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/32031248/d005bba4-16f9-4971-bc3f-3144c913c473)